Sources_v3/StreamChat/Database/DataStore.swift

```diff
@@ -31,6 +31,8 @@ public struct DataStore<ExtraData: ExtraDataTypes> {
     /// - Returns: If there's a user object in the locally cached data matching the provided `id`, returns the matching
     /// model object. If a user object doesn't exist locally, returns `nil`.
     ///
+    /// **Warning**: Can be called on the `main` thread only.
+    ///
```

Comment:
_should_ be called

Reply:
fixed in 0b20b8ae9b15ed9f6bda42e5fa2919c5e187a5b6